<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYConvInitialDockWidget</class>
 <widget class="QDockWidget" name="QPANSOPYConvInitialDockWidget">
   <property name="minimumSize">
    <size>
      <width>250</width>
      <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
      <width>16777215</width>
      <height>16777215</height>
    </size>
   </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Input Layer</string>
      </property>
      <layout class="QVBoxLayout" name="inputLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QgsMapLayerComboBox" name="routingLayerComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
             <item>
                <layout class="QFormLayout" name="formLayoutParams">
                 <property name="labelAlignment">
                    <set>Qt::AlignRight|Qt::AlignVCenter</set>
                 </property>
                 <item row="0" column="0">
                    <widget class="QLabel" name="labelProcAlt">
                     <property name="text">
                        <string>Procedure Altitude (ft)</string>
                     </property>
                    </widget>
                 </item>
                 <item row="0" column="1">
                    <widget class="QDoubleSpinBox" name="procAltSpinBox">
                     <property name="maximum">
                        <double>100000.000000000000000</double>
                     </property>
                     <property name="decimals">
                        <number>1</number>
                     </property>
                     <property name="value">
                        <double>1000.000000000000000</double>
                     </property>
                    </widget>
                 </item>
                 <item row="1" column="0">
                    <widget class="QLabel" name="labelMoc">
                     <property name="text">
                        <string>MOC</string>
                     </property>
                    </widget>
                 </item>
                 <item row="1" column="1">
                    <layout class="QHBoxLayout" name="mocLayout">
                     <item>
                        <widget class="QDoubleSpinBox" name="mocSpinBox">
                         <property name="maximum">
                            <double>100000.000000000000000</double>
                         </property>
                         <property name="decimals">
                            <number>1</number>
                         </property>
                         <property name="value">
                            <double>300.000000000000000</double>
                         </property>
                        </widget>
                     </item>
                     <item>
                        <widget class="QComboBox" name="mocUnitComboBox">
                         <item>
                            <property name="text">
                             <string>ft</string>
                            </property>
                         </item>
                         <item>
                            <property name="text">
                             <string>m</string>
                            </property>
                         </item>
                        </widget>
                     </item>
                    </layout>
                 </item>
                </layout>
             </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outputGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Output Options</string>
      </property>
      <layout class="QVBoxLayout" name="outputLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="exportKmlCheckBox">
         <property name="text">
          <string>Export to KML</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="folderLayout">
         <item>
          <widget class="QLineEdit" name="outputFolderLineEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>Browse</string>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="actionGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QVBoxLayout" name="actionLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="text">
          <string>Calculate CONV Initial Areas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
      <item>
       <widget class="QGroupBox" name="logGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
          </sizepolicy>
         </property>
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="logLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumHeight">
        <number>100</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
       </item>
      </layout>
     </widget>
    </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
      <width>20</width>
      <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
